daily-shoutout/app.py

```diff
@@ -55,11 +56,22 @@ def sendTweet(tweet_string):
     client.create_tweet(text = tweet_string)
 
 
+def returnResponse(response_string):
+    return{
+        "statusCode": 200,
+        "body": json.dumps(response_string)
+    }
+
+
 def lambda_handler(event, context):
     person = people[get_random_index(len(people))]
     cause = causes[get_random_index(len(causes))]
```

Comment:
Check out [`random.choice`](https://docs.python.org/3/library/random.html#random.choice) for a more concise form of this:

```suggestion
    person = random.choice(people)
    cause = random.choice(causes)
```

Reply:
Will do